pkg/apis/kube-scheduler/v1alpha1/types.go

```diff
@@ -23,7 +23,15 @@ type KubeScheduler struct {
 
 type KubeSchedulerSpec struct {
 	// Fill me
+	Kubeconfig string `json:"kubeconfig"`
+	PolicyConfigFile string `json:"policyconfigfile"`
```

Comment:
The above should be configmap name i think.

Reply:
This field is deprecated but this is what we are using in 3.11